scripts/BeforeBuild.ps1

```diff
@@ -16,8 +16,6 @@
     Do not restore NuGet packages.
 .PARAMETER NoObjectModel
     Do not rebuild the SARIF object model from the schema.
-.PARAMETER NoBuildSample
```

Comment:
>NoBuildSample [](start = 11, length = 13)

We have too many knobs in our build script and no easy ways to do common scenarios such as (please just rebuild and run tests rather than everything else). We also don't succeed in Debug configurations.